codespell_lib/data/dictionary.txt

```diff
@@ -32811,6 +32816,8 @@ resposive->responsive
 resposiveness->responsiveness
 resposne->response
 resposnes->responses
+respponse->response
```

Comment:
How about adding `respponding->respponding`, and perhaps other variants?

Reply:
I couldn't find such typos in GitHub, and decided not to include them.

Reply:
> I couldn't find such typos in GitHub, and decided not to include them.

Makes sense to me.